koku/api/apps.py

```diff
@@ -14,8 +14,70 @@
 #    You should have received a copy of the GNU Affero General Public License
 #    along with this program.  If not, see <https://www.gnu.org/licenses/>.
 #
+"""API application configuration module"""
+
+import logging
+
 from django.apps import AppConfig
 
+from koku.env import ENVIRONMENT
+
+
+logger = logging.getLogger(__name__)  # pylint: disable=invalid-name
+
 
 class ApiConfig(AppConfig):
+    """API application configuration"""
+
     name = 'api'
+
+    def ready(self):
+        """Action on application startup."""
+        # noqa: E402 pylint: disable=C0413
+        from django.db.utils import OperationalError
+        try:
+            self.startup_status()
+            self.check_and_create_service_admin()
+        except OperationalError as op_error:
+            logger.error('Error: %s.', op_error)
+
+    def startup_status(self):  # pylint: disable=R0201
+        """Log the status of the server at startup."""
+        # noqa: E402 pylint: disable=C0413
+        from api.status.model import Status
+        status_info = None
+        status_count = Status.objects.count()
+        if status_count == 0:
+            status_info = Status.objects.create()
+            status_info.save()
+        else:
+            status_info = Status.objects.get(pk=1)
```

Comment:
If we have more than instance of the API running ever this function may lie to us. It's a generated UUID, so I'm uncertain how we're tying it to the server anyway. 

Also, with Postgres the PK is likely a sequence which doesn't auto-reset to 1 if we end up wiping (or creating multiple Status objects--like might happen when testing). This can fail if the sequence increments and our status object has say pk=2. `Status.objects.all().first()` would just get the first entry regardless of pk. However, see the above comment about storing multiple servers.

Reply:
I'll use `first()`. We can evaluate whether the multi-instance issue has any impact. At this point the server uuid doesn't mean a lot to us. We could even consider dropping it and just never storing the object in the db.

Reply:
If we don't plan on adding more columns to the Status model/table, or a way to tie it directly to hardware/a VM/a container??, I vote for dropping it. Not in this story, but in a follow up issue.